don't add webview to STD_GUI_LIBS to avoid it being picked up by default
by other ports.

Index: configure
--- configure.orig
+++ configure
@@ -37595,7 +37595,7 @@ if test "$wxUSE_GUI" = "yes"; then
         BUILT_WX_LIBS="stc $BUILT_WX_LIBS"
     fi
     if test "$wxUSE_WEBVIEW" = "yes" ; then
-        STD_GUI_LIBS="webview $STD_GUI_LIBS"
+        # STD_GUI_LIBS="webview $STD_GUI_LIBS"
         BUILT_WX_LIBS="webview $BUILT_WX_LIBS"
     fi
     if test "$wxUSE_XRC" = "yes" ; then
@@ -39051,6 +39051,11 @@ fi
         USE_SOVERSION=1
         USE_SOVERLINUX=1
         USE_SOTWOSYMLINKS=1
+        case "${BAKEFILE_HOST}" in
+          *-*-openbsd*)
+            USE_SOTWOSYMLINKS=0
+          ;;
+        esac
       ;;
 
       *-*-solaris2* )
